pages/admin/event/event.js

```diff
@@ -181,16 +181,16 @@ function createEvent(e) {
     }, {
         success: function (result) {
           that.setData({loading: false})
-          common.showTip('Success');
           wx.navigateBack({
             delta: 1
           })
+          common.showTip('Success');
```

Comment:
It is to ensure "Success" message can display on screen after navigating back to admin page